GOVERNANCE.md

```diff
@@ -0,0 +1,221 @@
+- If a proposal cannot be resolved by the affected teams, the decision may be escalated to the Org team.
+
+## Voting
+
+- Most team decisions are made by lazy consensus <https://communitymgt.wikia.com/wiki/Lazy_consensus>.
+- In these cases, if no consensus can be reached, the matter may be resolved by Simple Majority Vote <https://en.wikipedia.org/wiki/Majority> by the appropriate team.
+- However there are cases that always require a Simple Majority or Supermajority Vote <https://en.wikipedia.org/wiki/Supermajority> by the appropriate team.
+These are defined below.
+
+### Simple Majority Changes
+
+- Org team: Licensing and intellectual property changes.
+- Org team: Using Flux/CNCF money for anything CNCF deems "not cheap and easy".
+
+### Supermajority Changes
+
+- Org team: Enforcing a Code of Conduct violation.
+- Org team: Removing a member from a team for any reason other than inactivity.
+- Org team: Material changes to the Governance document.
+  - Editorial changes to governance may be made by Org team lazy consensus, unless challenged.
+These are changes which fix spelling or grammar, update work affiliation or similar, update style or reflect an outside and obvious reality.
+They do not change the intention or meaning of anything in this document.
+
+### Membership Changes
+
+- All team members MUST remain active contributors.
+If they are unresponsive for > 3 months they will be automatically removed unless a supermajority of the other members of that team agrees to extend the period.
+- A member may voluntarily step down from a team by following the proposal process in the community repo.
+These proposals are automatically accepted.
+Once offboarding is complete and the issue is closed, they become an emeritus member of that team.
+- New members may be added to a team by a supermajority vote by the existing members of that team.
+- When a team has no members the Org team become responsible for it and may either archive the team or find new members.
+- When a team is archived, its members become emeritus members of that team.
+
+Org team membership changes have additional considerations:
+
+- Members of any other team are eligible to be nominated as an Org team member.
+- Org maintainers MUST remain current members of other non-archived teams.
+If that status changes, they will also loose Org team membership.
+- If an Org maintainer volunteraily steps down, in addition to the process above, within 7 calendar days the Flux dev list MUST be notified <https://lists.cncf.io/g/cncf-flux-community>.
+This gives contributors reasonable time to be made aware of the change.
+- When there is an opening for a new Org maintainer, any contributor to a repository in the `fluxcd` GitHub org may nominate a suitable existing member of another team as a replacement.
```

Comment:
This suggests there's a limited number of Org maintainers, or a process for deciding when there are spots free. That ought to be documented too.

Reply:
I first thought this too, but later removed my (pending) comment when I came to the conclusion that the interpretation can be very wide. Technically speaking there can be an opening if the current Org maintainers deem it necessary (like a job opening).

However, given we both had this thought, it likely requires some additional documentation :-).

Reply:
I think you're both right we should clarify this. For example, Helm's Org group states:
> There will be between 3 and 9 people

Do you have any thoughts about what would work best for Flux?

Reply:
3 as a minimum makes sense to not become authoritarian or end up in a 50/50 conflict. I do not have an exact maximum in mind, except that my experience is that "more isn't always better".

Reply:
I pushed some changes based on your comments. WDYT?

Reply:
This is out of date with current simplifications.